packages/conventional-changelog-preset-loader/index.js

```diff
@@ -35,6 +35,10 @@ function presetLoader (requireMethod) {
       // a promise.
       if (config && !config.then && typeof path === 'object') {
         return config(path)
```

Comment:
Rather than the additional condition, should we just alter the condition that determines whether or not it's a builder?

```js
if (config && !config.then) {
  const options = typeof path === 'object' ? path : {}
  return config(options)
}
```

Reply:
Yes, that looks much better. Thanks.